src/Reference/Task2.java

```diff
@@ -0,0 +1,26 @@
+package Reference;
+
+import java.util.Scanner;
+
+public class Task2 {
+    public static void main(String[] args) {
+
+        Scanner reader = new Scanner(System.in);
+        System.out.println("Enter dimension of array: ");
+        int dim = reader.nextInt();
+
+        int[] array = new int[dim];
+
+        for (int i = 0; i < dim; i++){
+            System.out.println("Enter " + i + " element");
+            array[i] = reader.nextInt();
+        }
```

Comment:
If we decide to close our readers explicitly (Task1), then let's be consistent and close it wherever we use it)

Reply:
Ok